rules/autoconsent/xhamster-us.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "xhamster-us",
+    "cosmetic": true,
```

Comment:
Since this is meant for one site only, shall we use the `urlPattern` here? Check the pornhub rule for example.

Reply:
Note, I ran a quick US top 10k crawl on this branch, and this rule also matches `https://xhamster2.com/` and `https://xhamster3.com/`.

Reply:
In this case, I think something like `\/\/[^/]*xhamster` should be good enough